python/src/scipp/_arithmetic.py

```diff
@@ -0,0 +1,158 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+
+from __future__ import annotations
+
+from ._scipp import core as _cpp
+from ._cpp_wrapper_util import call_func as _call_cpp_func
+from .typing import DataArrayLike
+
+
+def add(a: DataArrayLike, b: DataArrayLike) -> DataArrayLike:
+    """Element-wise addition.
+
+    Equivalent to
+
+    .. code-block:: python
+
+        a + b
+
```

Comment:
Is it worth referring to this https://scipp.github.io/user-guide/computation.html ?

Reply:
Good idea